doc/modules/preprocessing.rst

```diff
@@ -480,6 +480,39 @@ values than observed values.
 :class:`Imputer` can be used in a Pipeline as a way to build a composite
 estimator that supports imputation. See :ref:`sphx_glr_auto_examples_missing_values.py`
 
+.. _missing_indicator:
+
+Transformer indicating missing values
+=====================================
```

Comment:
this belongs as a subheading under "imputation of missing values", IMO, not an equal heading.